packages/components/sorts/HighVolSortButton.tsx

```diff
@@ -0,0 +1,74 @@
+import React from "react";
+import { StyleProp, TouchableOpacity, View, ViewStyle } from "react-native";
+
+import chevronDownSVG from "../../../assets/icons/chevron-down.svg";
+import chevronUpSVG from "../../../assets/icons/chevron-up.svg";
+import sortSVG from "../../../assets/icons/sort.svg";
+import { SortDirection } from "../../api/marketplace/v1/marketplace";
+import { neutral11, secondaryColor } from "../../utils/style/colors";
+import { fontSemibold14 } from "../../utils/style/fonts";
+import { BrandText } from "../BrandText";
+import { SVG } from "../SVG";
+import { Box } from "../boxes/Box";
+
+export const HighVolSortButton: React.FC<{
+  sortDirection: SortDirection;
+  onChangeSortDirection: (val: SortDirection) => void;
+  height?: number;
+  style?: StyleProp<ViewStyle>;
+}> = ({ style, sortDirection, onChangeSortDirection, height = 48 }) => {
+  const handlePress = () => {
+    if (sortDirection === SortDirection.SORT_DIRECTION_DESCENDING) {
+      onChangeSortDirection(SortDirection.SORT_DIRECTION_ASCENDING);
+    } else {
+      onChangeSortDirection(SortDirection.SORT_DIRECTION_DESCENDING);
+    }
+  };
+
+  return (
+    <TouchableOpacity onPress={handlePress} style={style}>
+      <Box
+        notched
+        style={[
+          {
+            borderWidth: 1,
+            borderColor: "#FFFFFF",
+            paddingHorizontal: 13,
```

Comment:
For all padding / margin, you can use layout.spacing_x ....

Reply:
Fixed, PR updated